src/tests/end2end/ComputeCopyStorageBufferTests.cpp

```diff
@@ -145,11 +148,14 @@ TEST_P(ComputeCopyStorageBufferTests, DISABLED_SizedArray) {
 }
 
 // Test with an unsized array SSBO.
+//
+// TODO(kainino@chromium.org): This test may be somewhat wrong. I'm not sure whether this is
+// supposed to be possible on the various native APIs.
+// Linking on OpenGL fails with "OpenGL requires constant indexes for unsized array access(dst)".
```

Comment:
I think the unsized array has to be the last member of a structure that's the root of the SSBO. 

```
layout(std140, set=0, binding=0) buffer Src {S s[];} src;
```

Reply:
This test would be an unsized array of descriptors, not a single descriptor referring to an unsized buffer. Again, I'm not sure it's possible.

I should probably add a test for an unsized buffer though.

Reply:
Updated the test descriptions to be more accurate (now that I understand what they should be doing).

Reply:
Also updated test names and added the unsized buffer test.